lib/hammer_cli_foreman_tasks/recurring_logic.rb

```diff
@@ -6,9 +6,15 @@ class ListCommand < HammerCLIForeman::ListCommand
       output do
         field :id, _('ID')
         field :cron_line, _('Cron line')
+        field :task_count, _('Task count')
+        field :action, _('Action')
+        field :last_occurence, _('Last occurrence')
+        field :next_occurence, _('Next occurrence')
```

Comment:
For fields with datetime data it's better to use `Fields::Date` formatter.

Reply:
Good point, done